docs/management_api.yml

```diff
@@ -884,8 +884,10 @@ definitions:
     properties:
       name:
         type: string
+        description: Name of the deployment
```

Comment:
Maybe lets decide whether we use the articles (a/an/the) in the descriptions or not?

Reply:
I have the impression we already decided to NOT use articles in descriptions, and more or less this is what we have all across the different specification files. What do you think?

Reply:
I have no problem with no articles as long as it is consistent. no strong opinions, since I am not a native english speaker :) Spotify seems to be using articles e.g.: [The Spotify ID for the artist](https://developer.spotify.com/documentation/web-api/reference/artists/get-artist/) and Slack as well: [A unique ID for the incident](https://api.slack.com/docs/slack-status#v2_0_0)